src/Systemd/Journal.hsc

```diff
@@ -439,7 +439,7 @@ openJournal flags start journalFilter threshold =
         go journalPtr
 
       EQ -> when (sdJournalDirection == Forwards) $ do
-        liftIO $ sdJournalWait journalPtr (-1)
+        liftIO $ sdJournalWait journalPtr maxBound
```

Comment:
What is this change?

Reply:
this includes the changes from Erik's PR #20 that fixes a compiler warning about a type overflow.

Reply:
I was not able to cherry-pick this in the fork.

Reply:
Aha, yes. I'll merge that separately - thanks!